SensorFusion_src/main_test.c

```diff
@@ -0,0 +1,51 @@
+/*******************************************************
+* Author : Shyam Bhuptani 
+* Function : Unit Testing of all Functions
+*/
+
+/*Including all dependancies*/
+#include <stdio.h>
+#include <stdlib.h>
+#include "../include/support_degree_matrix.h"
+#include "../include/contribution.h"
+#define num1 2
```

Comment:
Not a good variable name. num1 doesn't mean anything to me.

Reply:
Updated in new commit, Please check